great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -118,7 +118,13 @@ def get_rules(self) -> Optional[List[Rule]]:
             max_unexpected_ratio=None,
             min_max_unexpected_values_proportion=9.75e-1,
         )
-        numeric_columns_rule: Rule = self._build_numeric_columns_rule()
+        numeric_columns_low_precision_rule: Rule = (
+            self._build_numeric_columns_low_precision_rule()
+        )
+        numeric_columns_high_precision_rule: Rule = (
+            self._build_numeric_columns_high_precision_rule()
+        )
+
```

Comment:
@NathanFarmer Let us delete this blank line (style consistency).

Reply:
✅